.github/workflows/dependencies_pr.yml

```diff
@@ -0,0 +1,27 @@
+name: Open dependencies PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        description: "New semantic version string"
+        required: true
+
+jobs:
+  dep_pr:
+    steps:
+      - name: Permissions Check
+        uses: ./.github/workflows/actions/permission_check
+        with:
+          token: ${{ secrets.FPPF_BOT_AUTH_KEY }}
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.FPPF_BOT_AUTH_KEY }}
+      - name: Open develop_dependencies PR
+        if: ${{ inputs.open_dependencies_pr && !inputs.draft }}
```

Comment:
Where are the `open_dependencies_pr` and `draft` input defined?

Reply:
If you go to the actions tab and click on eg bluemira_examples there is a box on the rhs

Reply:
I can see them in release.yml but surely they have to be defined as an input for this composite action?